tests/_files/source_with_multiline_constant_return.php

```diff
@@ -245,22 +245,7 @@ public function Spaceship(): int
         ;
     }
 
-    public function nowdocSimpleA(): string
-    {
-        return <<<'EOF'
-            foo
-            EOF;
-    }
-
-    public function nowdocSimpleB(): string
-    {
-        return
-            <<<'EOF'
-                foo
-                EOF;
-    }
-
-    public function nowdocSimpleC(): string
+    public function nowdoc(): string
```

Comment:
heredoc/nowdoc with 2+ lines content should be tested to assert startLine + 1 is correct